lib/cldr/territory.ex

```diff
@@ -156,6 +156,47 @@ defmodule Cldr.Territory do
     module.known_territory_subdivisions(territory_code)
   end
 
+  @doc """
+  Reurns the display name for the given territory code.
+  Returns `{:ok, String.t}` if successful, otherwise `{:error, reason}`.
+
+  * `options` are:
+    * `locale` is any configured locale. See `Cldr.known_locale_names/1`.
+      The default is `Cldr.get_locale/1`
+
+    * `backend` ia any Cldr backend module. The default is
```

Comment:
```suggestion
    * `backend` is any Cldr backend module. The default is
```

Reply:
Thanks for the review. Sloppy on my part. Fixed both and pushed now.